tests/ballerina-test-integration/src/test/java/org/ballerinalang/test/service/websub/WebSubIntegrationTestCase.java

```diff
@@ -74,7 +74,7 @@ public void setUp() throws BallerinaTestException, InterruptedException {
         });
 
         //Allow to bring up the hub
-        Thread.sleep(15000);
+        Thread.sleep(10000);
```

Comment:
This kind of sleep will result in intermittent issues. It is better to use Awaitility instead of Thread.sleep. There are samples in the codebase where Awaitility has been used.